evaluators/embedding/MinkowskiDistanceEvaluator/__init__.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+
+from jina.executors.evaluators.embedding import BaseEmbeddingEvaluator, expand_vector
+
+
+class MinkowskiDistanceEvaluator(BaseEmbeddingEvaluator):
+    """A :class:`MinkowskiDistanceEvaluator` evaluates the distance between actual and desired 
+    embeddings computing the Minkowski distance ( p>0 ) between them, which can be considered 
+    as a generalization of both the Euclidean distance and the Manhattan distance.
+
+        D(x, y) = ( sum_i |x_i - y_i|^p )^(1/p)
+    """
+
+    @property
+    def metric(self):
+        return 'MinkowskiDistanceEvaluator'
+
+    def evaluate(self, actual: 'np.array', desired: 'np.array', *args, **kwargs) -> float:
+        """"
+        :param actual: the embedding of the document (resulting from an Encoder)
+        :param desired: the expected embedding of the document
+        :return the evaluation metric value for the request document
+        """
+        actual = expand_vector(actual)
+        desired = expand_vector(desired)
+        order = kwargs["order"]
```

Comment:
this order will not come here, the `order` must be an `attribute` of the class, but will not come in this `evaluate` `kwargs`. We may need to remove the `variable arguments` from the signatures.

Reply:
Are you meaning that the order should be an attribute of the class or should be an attribute of an instance?
My understanding is that this is a general evaluator which accepts any positive p as its order. Therefore it might be an attribute of an instance.

Reply:
yes, it would be an attribute of an instance sorry

Reply:
Hey @smy0428 would you please fix this and make it an instance attribute?